Makefile

```diff
@@ -33,7 +33,7 @@ collectstatic:
 
 dtest = docker-compose -f docker-compose.test.yml 
 test:
-	$(dtest) build && $(dtest) run --rm test-web pytest
+	$(dtest) build --build-arg INSTALL_DEV=--dev && $(dtest) run --rm test-web pytest
```

Comment:
Naaaah not a big fan - this still allows any shit to be sent, and now you also have to duplicate it `INSTALL_DEV` and also give `--dev`. Don't pipe the argument through - check it in the dockerfile and if something is given, then install.